app/assets/scripts/components/connected/user-menu.js

```diff
@@ -9,14 +9,40 @@ import { logoutUser } from '../../actions';
 import Dropdown from '../dropdown';
 
 class UserMenu extends React.Component {
+  constructor () {
+    super();
+    this.state = {
+      width: window.innerWidth
+    };
+  }
+
+  // Adds a listener for window size to determine style for menu content
+  componentWillMount () {
+    window.addEventListener('resize', this.handleWindowSizeChange);
```

Comment:
I am slightly concerned that this might get called a lot as the user is resizing the window. While testing though and trying to resize a lot, I did not notice any weird behaviour, so I'm going to go ahead and merge. We may want to add something like a `debounce` to the function in the future: https://lodash.com/docs/4.17.15#debounce

Reply:
That's a fair concern. It's not ideally set up for users to resize the window. I don't imagine that users will resize fairly often, but I will try to re-jig this in a new pr so that it won't come up as an issue.